packages/cli/src/command.js

```diff
@@ -48,9 +48,12 @@ export default class NuxtCommand {
 
     const runResolve = Promise.resolve(this.cmd.run(this))
 
-    // TODO: For v3 set timeout to 0 when force-exit === true
-    if (!this.isServer || this.argv['force-exit']) {
-      runResolve.then(() => forceExit(this.cmd.name, forceExitTimeout))
+    if (
+      this.argv['force-exit'] ||
+      // dont force exit when it was explicitly disabled by the user
+      (!this.isServer && !this.isExplicitArgument('force-exit'))
```

Comment:
Why `!this.isServer`? 🤔

Reply:
Its to differentiate between build/generate and dev/start without having the need to change the default value of force-exit